network-arbitrary.cabal

```diff
@@ -24,72 +31,85 @@ extra-source-files:
   , README.md
   , Setup.hs
 
-source-repository head
-  type:     git
-  location: https://github.com/alunduil/network-arbitrary
-  branch:   develop
-
 library
-  default-language:    Haskell2010
+  ghc-options:
```

Comment:
is -Wall going to be overwritten by if?

Reply:
Nope, duplicate sections in cabal files are concatenated together so this is the correct way to describe this and have a default augmented by the if statement.

Reply:
If you want to see this in action you might find it in the travis output attached to this pull request.